src/vector.jl

```diff
@@ -62,20 +105,19 @@ end
 
 # Arithmetic
 for (jf, af) in [(:+, :add!), (:-, :sub!)]
-    @eval function Base.$jf(v::T, w::T) where {T<:Union{ArbVector,AcbVector}}
+    @eval function Base.$jf(v::T, w::T) where {T<:Vectors}
```

Comment:
these functions does not even work on the syntactic level ;)
but it's too much work to fix it here, I already did in #56